datatypes/mcs_int128.h

```diff
@@ -265,6 +265,13 @@ class TSInt128
       return toTFloat128();
     }
 
+    //    unaligned argument
+    inline TSInt128& operator=(const int128_t* x)
```

Comment:
This assignment operator looks a bit odd IMHO but TSInt128 is our class and we can have whatever semantics we need.